website/data/docs-navigation.js

```diff
@@ -7,183 +7,98 @@
 //   serve as the category title in the sidebar
 
 export default [
-  { category: 'install', content: ['ports', 'bootstrapping', 'performance'] },
   {
-    category: 'upgrading',
-    content: ['compatibility', 'upgrade-specific'],
-  },
-  'glossary',
-  {
-    category: 'internals',
-    content: [
-      'architecture',
-      'consensus',
-      'gossip',
-      'coordinates',
-      'sessions',
-      'anti-entropy',
-      'security',
-      'jepsen',
-      'discovery-chain',
-    ],
-  },
-  {
-    category: 'commands',
+    category: 'intro',
     content: [
       {
-        category: 'acl',
+        category: 'vs',
+        name: 'Consul vs. Other Software',
         content: [
-          {
-            category: 'auth-method',
-            content: ['create', 'delete', 'list', 'read', 'update'],
-          },
-          {
-            category: 'binding-rule',
-            content: ['create', 'delete', 'list', 'read', 'update'],
-          },
-          'bootstrap',
-          {
-            category: 'policy',
-            content: ['create', 'delete', 'list', 'read', 'update'],
-          },
-          {
-            category: 'role',
-            content: ['create', 'delete', 'list', 'read', 'update'],
-          },
-          'set-agent-token',
-          {
-            category: 'token',
-            content: ['clone', 'create', 'delete', 'list', 'read', 'update'],
-          },
-          'translate-rules',
+          'zookeeper',
+          'chef-puppet',
+          'nagios-sensu',
+          'skydns',
+          'smartstack',
+          'serf',
+          'eureka',
+          'istio',
+          'proxies',
+          'custom',
```

Comment:
There are a few additional changes we need to make here:

- All of this content previously existed at `/intro/vs`, and still does. Duplicate content at different routes is not allowed so that we do not confuse users and get downranked by search engines. My recommendation here would be to delete the `/intro/vs` folder, and add redirects for each retained route to its new `/docs/vs` destination.
- This category previously had an overview page [which can be seen here](https://www.consul.io/intro/vs) - in this version the use of `name` indicates that the intention is to remove it. This is ok, if it was a conscious decision, but we should to have a redirect in place for any links to this page to avoid broken links. We also should remove the actual page, which is still retained as `index.mdx` but not displayed in the nav. Alternately, we can remove the `name` and just adjust the frontmatter for the index page if the desire was to change the category name.

And finally, this isn't a fix we need to make as much as an observation, but it may also be worth considering whether we want to retain this section at all -- in general "ours vs theirs" sections written by the company who makes one of the solutions tend to be extremely biased and not looked upon well. Vault recently entirely removed the "vs" section in their docs. It is of course possible to write in a useful way by describing how related technologies are different, but must be approached very carefully.